A6lib.cpp

```diff
@@ -441,3 +441,31 @@ byte A6lib::A6waitFor(const char *resp1, const char *resp2, int timeout, String
     }
     return retVal;
 }
+
+
+// making modem functioning or free from previous engagements
+int A6lib::getFree(long timeout)
```

Comment:
Oh, that sounds useful to have, it's just that the comment doesn't really explain the above. If you could write the above explanation somewhere around the function declaration in a comment, it would be ideal.